iml-wasm-components/iml-action-dropdown/src/fetch.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use futures::Future;
```

Comment:
Take a look at this, it seems like a simpler way to do fetch: https://seed-rs.org/guide/5

Reply:
Ah nice, it looks like they've already implemented something similar. I'll use this instead.